.github/workflows/tests.yml

```diff
@@ -0,0 +1,166 @@
+        run: yarn install
+      - name: 'Model Tests'
+        env:
+          RAILS_ENV: test
+          DB_PASSWORD: root
+          DB_PORT: ${{ job.services.mysql.ports[3306] }}
+          DISABLE_BOOTSNAP: true
+        run: |
+          bundle exec rails test test:models
+
+  integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.4.6
+          bundler-cache: true
+      - uses: ./.github/actions/setup-test-environment
+      - uses: egordm/gha-yarn-node-cache@v1
+      - name: Install JavaScript dependencies with Yarn
+        run: yarn check || yarn install;
+      - name: 'Integration Tests'
+        env:
+          RAILS_ENV: test
+          DB_PASSWORD: root
+          DB_PORT: ${{ job.services.mysql.ports[3306] }}
+          DISABLE_BOOTSNAP: true
+        run: bundle exec rails test test:integration
+
+  controller-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.4.6
+          bundler-cache: true
+      - uses: ./.github/actions/setup-test-environment
+      - uses: egordm/gha-yarn-node-cache@v1
+      - name: Install JavaScript dependencies with Yarn
+        run: yarn check || yarn install;
+      - name: 'Controller Tests'
+        env:
+          RAILS_ENV: test
+          DB_PASSWORD: root
+          DB_PORT: ${{ job.services.mysql.ports[3306] }}
+          DISABLE_BOOTSNAP: true
+        run: bundle exec rails test test:controllers
+
+  system-tests:
+    runs-on: ubuntu-latest
+    services:
+      redis:
+        image: redis
+        # Set health checks to wait until redis has started
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps port 6379 on service container to the host
+          - 6379:6379
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.4.6
+          bundler-cache: true
+      - uses: nanasess/setup-chromedriver@v1.0.1
+      - uses: ./.github/actions/setup-test-environment
+      - uses: egordm/gha-yarn-node-cache@v1
+      - name: Install JavaScript dependencies with Yarn
+        run: yarn check || yarn install;
+      - name: 'System Tests'
+        env:
+          RAILS_ENV: test
+          DB_PASSWORD: root
+          DB_PORT: ${{ job.services.mysql.ports[3306] }}
+          REDIS_HOST: localhost
+          REDIS_PORT: 6379
+        run: |
+          export DISPLAY=:99
+          chromedriver --url-base=/wd/hub &
+          bundle exec rails test test:system
+      - name: Archive system test screenshots
+        uses: actions/upload-artifact@v2
+        with:
+          name: system-test-screenshots
+          path: tmp/screenshots/*
+
+  docker-development:
```

Comment:
Can we rename this? What is its role? It looks good but even familiar with MapKnitter and docker, it's not super clear what the purpose of this job is, so i wonder if we could explain it a bit using the name? Thanks @aliciapaz!!!

Reply:
Hmm, could it be `development-docker-build` and `production-docker-build`?
From what I understand, these jobs ensure that changes made to the source code do not break those docker builds. Does that make sense? 
Thank you @jywarren!